spec/datadog/tracing/configuration/http_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'spec_helper'
+
+require 'datadog/tracing/configuration/http'
+require 'datadog/core/utils/hash'
+
+RSpec.describe Datadog::Tracing::Configuration::HTTP::HeaderTags do
+  subject(:http_header_tags) { described_class.new(header_tags) }
```

Comment:
I find this `subject` a little confusing. It looks like the real subjects are the ones defined for the shared examples.

Suggestion, should we use a let instead here?

Reply:
Moved to `let`! 👍